schemachange/schemachange.c

```diff
@@ -534,76 +534,89 @@ int live_sc_post_delete(struct ireq *iq, void *trans, unsigned long long genid,
                         const void *old_dta, unsigned long long del_keys,
                         blob_buffer_t *oldblobs)
 {
-
+    int rc = 0;
+    pthread_rwlock_rdlock(&sc_live_rwlock);
```

Comment:
I prefer to wrap in function so we don't have to goto unlock. Following is more robust and easier to read:
```
int live_sc_post_delete() {
    lock()
    live_sc_post_delete_int() // this function is now _int
    unlock()
}
```

Reply:
sure, thanks